app/views/report/_workflow_table.html.erb

```diff
@@ -1,26 +1,28 @@
 <h2>Workflow: <%= render_workflow_name(wf_name) %></h2>
-<table border="1" cellspacing="0" cellpadding="3">
+<table class='table table-striped table-hover workflow-grid-table'>
```

Comment:
how about adding `table-condensed`? in production there's lots of data in the workflow grid...